src/utils/size-helpers.js

```diff
@@ -78,12 +75,6 @@ export const sizeDefaultProps = {
   isSizeEdit: false
 };
 
-export const sizeFormAccordionPropTypes = {
-  ...sizePropTypes.size,
-  absolutePrice: PropTypes.number,
-  relativePrice: PropTypes.number
-};
-
 export const sizeFilterObj = () => {
   const arrToFilter = [];
 
```

Comment:
If you store file in specific folder, you don't need to use suffix in naming 
Especially the wrong suffix (we have separate folder for helper functions)

Reply:
fixed